### PR TITLE
borrow boosting

### LINKED ISSUE #8: Make a Booster that supports borrowing
https://github.com/Rari-Capital/compound-protocol/blob/fuse-final/contracts/RewardsDistributorDelegate.sol#L221

and

https://github.com/Rari-Capital/compound-protocol/blob/fuse-final/contracts/RewardsDistributorDelegate.sol#L271

https://github.com/Rari-Capital/compound-protocol/blob/fuse-final/contracts/RewardsDistributorDelegate.sol#L300

### PR DESCRIPTION
**Pull request by Joeysantoro**
https://github.com/fei-protocol/fuse-flywheel/issues/8

### CODE REVIEW

File: src/FuseFlywheelBorrowerCore.sol [hunk 1]
```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.10;
+
+import "flywheel/FlywheelCore.sol";
+
+contract FuseFlywheelCore is FlywheelCore {
```
Comment (RedVeil) [inline src/FuseFlywheelBorrowerCore.sol:6]: Can we rename this to FuseFlywheelBorrowCore ?

**Comment by Pczek** [comment]
If I have a list of flywheels, how could I differentiate best if a flywheel rewards the borrow or supplier side?

File: src/FuseFlywheelBorrowerCore.sol (hunk 1 above)
Comment (Joeysantoro) [reply src/FuseFlywheelBorrowerCore.sol:6]: sure

**Comment by peetzweg** [comment]
> > If I have a list of flywheels, how could I differentiate best if a flywheel rewards the borrow or supplier side?
> 
> What's the context of your need here? Is it better if we implemented a function to estimate the APY/APR on a specific user?

Working in the frontend with the current Flywheels in the past, I an issue to differentiate between a "Flywheel with Static Rewards" and "Flywheels with dynamic Rewards". There is flag or something to tell you what kind of FlywheelRewards it is. One has to check if a function selector exists or the deployed bytecode is the same. Which is very cumbersome and request intensive.

I see a similar problem with Flywheels either rewarding the supplier or the borrower side. How to check who the flywheel is rewarding?

**Pull request merged**